example/lib/screen/nav_bar/nav_bar_home_screen.dart

```diff
@@ -0,0 +1,55 @@
+import 'package:flutter/material.dart';
+import 'package:xayn_design/xayn_design.dart';
+
+import 'base_nav_bar_screen.dart';
+
+class NavBarHomeScreen extends BaseNavBarScreen {
+  static const routeName = '/navBarHomeScreen';
+
+  const NavBarHomeScreen({Key? key}) : super(key: key);
+
+  @override
+  _NavBarHomeScreenState createState() => _NavBarHomeScreenState();
+}
+
+class _NavBarHomeScreenState extends BaseNavBarScreenState<NavBarHomeScreen>
+    with NavBarConfigsMixin {
+  @override
+  String get screenTitle => 'Home screen';
+
+  @override
+  Color get screenBgColor => Colors.orange;
+
+  @override
+  List<Widget> buildExtraChildren() => [
+        buildBtn('Open reader mode', openReaderMode),
+      ];
+
+  @override
+  NavBarConfigs get navBarConfigs {
+    final search = NavBarItemIconButton(
+      svgIconPath: linden.assets.icons.search,
+      isSelected: false,
+      onPressed: replaceSearch,
+      key: keySearch,
+    );
+    final account = NavBarItemIconButton(
+      svgIconPath: linden.assets.icons.person,
+      isSelected: false,
+      onPressed: replaceAccount,
+      key: keyPerson,
```

Comment:
💡  I guess as a good practice it would be nice to bundle those configs in a single file and expose them as factories, so we can reduce code duplication:

```
navBarConfigs get navBarConfigs => [searchNavBtutton(replaceSearch), accountNavBtutton(replaceAccount), ...]
```

Reply:
well, agreed 😉 
this is just an example app, but yeah, can try

Reply:
fixed